glue/clients/ds9norm.py

```diff
@@ -123,8 +123,8 @@ def __init__(self):
         self.stretch = 'linear'
         self.bias = 0.5
         self.contrast = 1.0
-        self.clip_lo = 5.
-        self.clip_hi = 95.
+        self.clip_lo = 0.
+        self.clip_hi = 100.
```

Comment:
Moving the stretch to the options widget seems like a good idea. I remember considering this a few times and I ran into some implementation issue. But conceptually I think it makes sense, and maybe it's easier than I remember